changelogs/fragments/4651-compatibility-virtualmedia-resource-location.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - xcc_redfish_command - for compatibility with the virtual media resource location on managers or systems. (https://github.com/ansible-collections/community.general/pull/4651).
```

I'm not sure what exactly this is fixing? Is this about *improving* compatibility?
```suggestion
  - xcc_redfish_command - for compatibility with the virtual media resource location on managers or systems (https://github.com/ansible-collections/community.general/pull/4651).
```